integration-tests/api/__tests__/admin/product.js

```diff
@@ -1484,6 +1499,36 @@ describe("/admin/products", () => {
         })
       )
     })
+
+    it("deletes a values associated with deleted option", async () => {
+      const api = useApi()
+
+      const response1 = await api.get(
+        "/admin/products/test-product-with-variant",
+        {
+          headers: {
```

Comment:
suggestion: Move the headers to a const above the main describe to use it everywhere